scss/_side-navigation.scss

```diff
@@ -0,0 +1,238 @@
+.side-nav {
+  // scss-docs-start side-nav-css-vars
+  --#{$prefix}side-nav-min-width: #{$side-nav-min-width};
+  --#{$prefix}side-nav-max-width: #{$side-nav-max-width};
+  --#{$prefix}side-nav-bg: #{$side-nav-bg};
+  --#{$prefix}side-nav-border-width: #{$side-nav-border-width};
+  --#{$prefix}side-nav-border-color: #{$side-nav-border-color};
+  --#{$prefix}side-nav-item-min-height: #{$side-nav-item-min-height};
+  --#{$prefix}side-nav-item-icon-size: #{$side-nav-item-icon-size};
+  --#{$prefix}side-nav-item-spacing: #{$side-nav-item-spacing};
+  --#{$prefix}side-nav-item-padding-top: #{$side-nav-item-padding-top};
+  --#{$prefix}side-nav-item-padding-end: #{$side-nav-item-padding-end};
+  --#{$prefix}side-nav-item-padding-bottom: #{$side-nav-item-padding-bottom};
+  --#{$prefix}side-nav-item-padding-start: #{$side-nav-item-padding-start};
+  --#{$prefix}side-nav-item-color: #{$side-nav-item-color};
+  --#{$prefix}side-nav-item-hover-bg: #{$side-nav-item-hover-bg};
+  --#{$prefix}side-nav-item-active-bg: #{$side-nav-item-active-bg};
+  --#{$prefix}side-nav-item-active-border: #{$side-nav-item-active-border};
+  // scss-docs-end side-nav-css-vars
+
+  position: relative;
+  flex-shrink: 0;
+  min-width: add(var(--#{$prefix}side-nav-min-width), var(--#{$prefix}side-nav-border-width));
+  height: 100%;
+
+  :focus {
+    &[data-focus-visible-added] {
+      outline-offset: -$focus-visible-outer-width;
+      box-shadow: inset 0 0 0 add($focus-visible-inner-width, $focus-visible-outer-width) var(--#{$prefix}focus-visible-inner-color);
+    }
+  }
+
+  .accordion {
+    --#{$prefix}accordion-bg: transparent;
+    --#{$prefix}accordion-border-width: 0;
+    --#{$prefix}accordion-body-padding-top: 0;
+    --#{$prefix}accordion-body-padding-end: 0;
+    --#{$prefix}accordion-body-padding-bottom: 0;
+    --#{$prefix}accordion-body-padding-start: 0;
+
+    .accordion-button::after {
+      margin-right: var(--#{$prefix}side-nav-item-spacing);
+    }
+
+    .accordion-body {
+      --#{$prefix}side-nav-item-padding-start: calc(var(--#{$prefix}side-nav-item-icon-size) + var(--#{$prefix}side-nav-item-spacing) * 2 - var(--#{$prefix}side-nav-border-width)); /* stylelint-disable-line function-disallowed-list */
+
+      .accordion-body {
+        --#{$prefix}side-nav-item-padding-start: calc(var(--#{$prefix}side-nav-item-icon-size) * 2 + var(--#{$prefix}side-nav-item-spacing) * 3 - var(--#{$prefix}side-nav-border-width) * 2); /* stylelint-disable-line function-disallowed-list */
+      }
+    }
+  }
+
+  > * {
+    overflow: hidden;
+    background: var(--#{$prefix}side-nav-bg);
+  }
+
+  // Small trick to remove `.h-100` inside the markup
```

Comment:
Can't we just keep this already existing utility class? At least to reduce the size of the bundle?

Reply:
I'd say that in the doc there shouldn't be any utility class if a rule is already applied to the element. Maybe we need to have a talk soon since we are more experienced about this now !